src/parsers/urdf.hpp

```diff
@@ -31,6 +31,13 @@
 #include <exception>
 #include <limits>
 
+#ifdef WITH_HPP_FCL
```

Comment:
We discuss yesterday with @jcarpent to remove all #ifdef WITH ... from the header files and only keep then in the cpp files. Not to be done in this PR, just a side remark for later.